src/pages/sighting/StatusCard.jsx

```diff
@@ -22,6 +22,8 @@ export default function StatusCard({ sightingData }) {
   const dateString = dateCreated
     ? formatDate(dateCreated, true)
     : 'unknown date';
+  const detectionComplete = get(sightingData, 'stage') !== 'detection';
```

Comment:
Is there any 'stage' that could be an error? Is it necssarily true (and will it be forever) that !=='detection' will me that detection is complete?

Reply:
i dont actually know what all the stages even are! this statusCard is going to be fleshed out much more soon i think. for now we dont have the data to do much with it